.github/workflows/review-apps-delete-dangling.yml

```
@@ -0,0 +1,72 @@
+name: Delete dangling review apps
+
+on:
+  schedule:
+    - cron: "0 1 * * *" # Every night at 1am
+
+jobs:
+  cleanup_reviewapp:
+    name: Delete dangling review apps
+    runs-on: ubuntu-latest
+    env:
+      SPACE: int
+      CF_USERNAME: ${{ secrets.PaaSUsernameInt }}
+      CF_PASSWORD: ${{ secrets.PaaSPasswordInt }}
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install cf client
+      env:
+        CF_CLI_VERSION: v7
+      run: |
+        curl -L "https://packages.cloudfoundry.org/stable?release=linux64-binary&version=${CF_CLI_VERSION}" | tar -zx -C /tmp
+        sudo cp /tmp/cf7 /usr/local/bin/cf
+    - name: Retrieve list of active pull requests
+      run: |
+        # Gets open PR numbers and builds the review app name for each PR. Returns an array.
+        active_prs=(`curl -H "Accept: application/vnd.github.groot-preview+json" https://api.github.com/repos/UKGovernmentBEIS/beis-opss-psd/pulls | jq -r '.[] | "psd-pr-\(.number)"'`)
```

I think you can use the `$GITHUB_REPOSITORY` var here (in case we ever rename the repo or move it to a different org):

```suggestion
        active_prs=(`curl -H "Accept: application/vnd.github.groot-preview+json" https://api.github.com/repos/$GITHUB_REPOSITORY/pulls | jq -r '.[] | "psd-pr-\(.number)"'`)
```